lib/tree_stand/node.rb

```diff
@@ -154,15 +256,6 @@ def walk(&block)
       enumerator
     end
 
-    # @example
-    #   node.text # => "4"
-    #   node.parent.text # => "3 * 4"
-    #   node.parent.parent.text # => "1 + 3 * 4"
-    sig { returns(TreeStand::Node) }
-    def parent
-      TreeStand::Node.new(@tree, @ts_node.parent)
-    end
-
```

Comment:
Why remove parent?

Reply:
Hi Derek.
It was added in `THINLY_WRAPPED_METHODS`.